quipucords/api/source/serializer.py

```diff
@@ -353,6 +353,7 @@ def validate_hosts(hosts):
         relaxed_cidr_pattern = r'[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*\/[0-9]*'
         relaxed_invalid_ip_range = r'[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*-' \
                                    r'[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*'
+        relaxed_invalid_ip_range_alt = r'[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*-[0-9]*'
```

Comment:
Maybe we should check for a - in the other 3 groups.  For example: 1-2.56.6.3 would also be invalid.

Reply:
done